eva-accession-core/src/main/java/uk/ac/ebi/eva/accession/core/persistence/SubmittedVariantEntity.java

```diff
@@ -105,11 +105,14 @@ public SubmittedVariantEntity(Long accession, String hashedMessage, String assem
     }
 
     public ISubmittedVariant getModel() {
```

Comment:
Another thing to discuss: should we allow null flags in the constructor of this entity:

allow null flags: bad because if a non-strict process doesn't compute the flags, it will be assumed the flags were computed and the result was the default value.

not allowing null flags: bad because when we query in the WS if a variant is accessioned, we will have to fill in fake flags, because the constructor will need it but they won't be used. 

thoughts @pabarcgar  and @cyenyxe ?

Reply:
agreed to add another constructor that allows not specifying the defaults, to be used for querying, but the main constructor requires them